src/java.base/share/classes/java/security/cert/X509Certificate.java

```diff
@@ -581,9 +585,14 @@ public X500Principal getSubjectX500Principal() {
      * in the form "a1:a2:...:a8", where a1-a8 are hexadecimal values
      * representing the eight 16-bit pieces of the address. OID names are
      * returned as {@code String}s represented as a series of nonnegative
-     * integers separated by periods. And directory names (distinguished names)
+     * integers separated by periods. Directory names (distinguished names)
      * are returned in <a href="http://www.ietf.org/rfc/rfc2253.txt">
-     * RFC 2253</a> string format. No standard string format is
+     * RFC 2253</a> string format. otherNames are returned as a byte array
+     * containing the ASN.1 DER encoded form of the name, and may also return
+     * a third entry in the list containing its {@code type-id} in string
+     * format, and a fourth entry containing the ASN.1 DER encoding of
+     * its {@code value} without the context-specific constructed tag
```

Comment:
To be very specific, I would add that the 4th entry is a byte array, ex: "and a fourth entry as a byte array containing the ASN.1 DER encoding ..."

Reply:
Good idea. Fixed and updated the CSR as well.